site/content/agent-app-widgets/introduction.md

```diff
@@ -24,9 +26,10 @@ Widgets are primary elements of Agent App interface. There are number of ways yo
 
 ## Important notes
 
-To build Agent App Widgets you need to have basic knowledge about JavaScript and HTML. We've got a lot of sample apps ready for you to take further. If you got stuck at any point, feel free to contact us anytime via chat.
+From a technical point of view, an Agent App Widget is a JavaScript SDK. Therefore, you need to have some knowledge of HTML and JavaScript to build your own widget. 
```

Comment:
`Agent App Widget is a JavaScript SDK` - a widget is a web application made of HTML, CSS and Javascript. It may use the Javascript SDK to communicate with the Agent App. 😉

Reply:
done